src/@next/Switch/Switch.stories.tsx

```diff
@@ -29,6 +30,28 @@ const Template: Story = args => {
   );
 };
 
-export const Interactive = Template.bind({});
+export const Default = DefaultTemplate.bind({});
 
-Interactive.args = {};
+Default.args = {};
+
+const SwitchWithIconTemplate: Story = args => {
+  const [checked, setChecked] = useState(args.checked);
+
+  return (
+    <>
+      <div>The checkbox checked is {checked ? 'true' : 'false'}</div>
+      <div>
+        <Switch
+          {...args}
+          value={'switch'}
```

Comment:
The previous example you passed `checked` to `<Switch>`. Any concern why you don't pass it here?

Reply:
> @finalight don't forget to add tests

added

Reply:
removed the `checked={checked}` for both story